### PR TITLE
BXMSDOC-3993-master: Getting started with business processes for PAM 7.4

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc [hunk 1]
```diff
@@ -26,77 +26,95 @@ The *Package* specifies the location inside the existing project where the asset
...
-. Click on the business rules task and enter `Validation` in the *Name* field of the *Diagram properties* panel.
-. Expand *Implementation/Execution*, select *validation* in the *Rule Flow Group* field.
-. Define the following Java expression in the *On Exit Action* field :
+. Configure the business rule task:
+
+.. Click the business rules task.
+.. If the *Diagram properties* panel is not open, in the upper-right corner click the *Diagram properties* image:getting-started/diagram_properties.png[] icon.
+.. In the *Diagram properties* panel, enter `Validation` in the *Name* field.
+.. Expand *Implementation/Execution* and from the *Rule Flow Group* list select *validation*.
+.. In the *On Exit Action* field, enter the following Java expression:
```
Comment (pnagesha) [inline doc-content/enterprise-only/getting-started/business-process-proc.adoc]: Is it important to know that this is a Java expression?

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc [hunk 2]
```diff
@@ -26,77 +26,95 @@ The *Package* specifies the location inside the existing project where the asset
...
-. Scroll down and expand the *Data Assignments* section and click image:getting-started/btn_assign.png[] next to *Assignments*.
-. In the *Validation Data I/O* window, click *Add* to create the following assignments:
+.. Expand *Data Assignments* and click image:getting-started/btn_assign.png[] next to *Assignments*.
+.. In the *Validation Data I/O* window, click *Add* and create the following assignments:
 +
 image::getting-started/val-data-io.png[]
 
-. Click *Save*.
-. In the diagram editor, click *Save*, and *Save*, to confirm your changes.
+.. Click *Save*.
```
Comment (pnagesha) [inline doc-content/enterprise-only/getting-started/business-process-proc.adoc]: So click save 3 times?

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc [hunk 3]
```diff
@@ -26,77 +26,95 @@ The *Package* specifies the location inside the existing project where the asset
...
+. In the diagram editor, click *Save* and click *Save* again to confirm your changes.
 
 == Defining the validation data
-This section describes how to define the data that determines whether the application data is correct or contains an error or any missing information.
+This section shows you how to define the validation data that determines whether the application data is correct, contains an error, or has missing information.
```
Comment (pnagesha) [inline doc-content/enterprise-only/getting-started/business-process-proc.adoc]: Does the data distinguish between errors and missing information? Does it also tell you how to fix it?

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc [hunk 4]
```diff
@@ -26,77 +26,95 @@ The *Package* specifies the location inside the existing project where the asset
...
 
-. Click the created connection.
+. Configure the connection between the exclusive gateway and the business rule task:
+.. Click the connection.
 +
 image::getting-started/created-connection.png[]
+.. If the *Diagram properties* panel is not open, in the upper-right corner click the *Diagram properties* image:getting-started/diagram_properties.png[] icon.
+.. In the *Diagram properties* panel, enter `Invalid` in the *Name* field.
+.. Expand *Implementation/Execution* and select *Expression* in the *Condition Expression* section.
+.. From the drop-down list, select *drools* and enter `ValidationErrorDO()` in the *Condition Expression* field.
```
Comment (pnagesha) [inline doc-content/enterprise-only/getting-started/business-process-proc.adoc]: What is the name of the list? According to IBM SG p209 "Refer to a drop-down list by its label, followed by list." So we should be saying "From the <apple> list, select...."

File: assemblies/assembly_business-processes/main.adoc [hunk 5]
```diff
@@ -10,7 +10,7 @@ include::_artifacts/document-attributes.adoc[]
 include::_artifacts/author-group.adoc[]
 
 // Purpose statement for the assembly
-As a business rules and processes developer, you can use {CENTRAL} in {PRODUCT} to design business processes to meet specific business requirements. {PRODUCT} provides example projects with example business assets directly in {CENTRAL} as a reference. This document describes how to create and test an example mortgage application project based on the *Mortgage_Process* sample project included in {CENTRAL}.
+As a business rules and processes developer, you can use {CENTRAL} in {PRODUCT} to design business processes to meet specific business requirements. {PRODUCT} provides example projects with example business assets in {CENTRAL} as a reference. This document describes how to create and test an example mortgage application project based on the *Mortgage_Process* sample project included in {CENTRAL}.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply assemblies/assembly_business-processes/main.adoc]: Yes, this isn't clear. It is an improvement, I'd like to retain the relationship though. As in, the example business assets are part of the example projects and the example projects are available in {CENTRAL}. What do you think about:

"{PRODUCT} provides example projects in {CENTRAL} that contain examples of business assets."

File: doc-content/drools-docs/src/main/asciidoc/DMN/dmn-gs-creating-drd-proc.adoc [hunk 6]
```diff
@@ -10,15 +10,18 @@ image::dmn/dmn-traffic-violations-drd.png[]
...
-.. Click *Ok* to open the DMN asset in the DMN designer.
-. In the DMN designer canvas, drag two input nodes to the canvas. Double-click the input nodes and rename them as `Driver` and `Violation`.
-. Drag a decision node to the canvas. Double-click and rename it as `Fine`.
-. Click the *Violation* input node and select the `Create DMN Information Requirement` option to connect to the `Fine` decision node.
-. Drag another decision node to the canvas and double-click and rename it as `Should the driver be suspended?`.
-. Click the *Driver* input node and select the  `Create DMN Information Requirement` option to connect to the `Should the driver be suspended?` decision node.
-. Click the *Fine* decision node and select the  `Create DMN Information Requirement` option to connect to the `Should the driver be suspended?` decision node.
-. Click *Save* to open the *Confirm Save* dialog box and click *Save* again.
+. On the *traffic-violation* project's home screen, click *Add Asset*.
+. On the *Add Asset* page, click *DMN Decision*. The *Create new DMN* window is displayed.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/drools-docs/src/main/asciidoc/DMN/dmn-gs-creating-drd-proc.adoc]: Changed to "home page" and "window is opened".

File: assemblies/assembly_business-processes/main.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (csherrar) [reply assemblies/assembly_business-processes/main.adoc]: *adding "for reference purposes."

File: doc-content/drools-docs/src/main/asciidoc/DMN/dmn-gs-creating-drd-proc.adoc [hunk 7]
```diff
@@ -10,15 +10,18 @@ image::dmn/dmn-traffic-violations-drd.png[]
...
+.. In the *Create new DMN* window, enter `Traffic Violation` in the *DMN* name field.
+.. From the *Package* list, select `org.kie.example.traffic.traffic_violation`.
+.. Click *Ok*. The DMN asset in the DMN designer is displayed.
+. In the DMN designer canvas, drag two *DMN Input Data* input nodes to the canvas.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/drools-docs/src/main/asciidoc/DMN/dmn-gs-creating-drd-proc.adoc]: Done

File: doc-content/drools-docs/src/main/asciidoc/DMN/dmn-gs-creating-drd-proc.adoc [hunk 8]
```diff
@@ -10,15 +10,18 @@ image::dmn/dmn-traffic-violations-drd.png[]
...
+. Double-click each input node and rename one to `Driver` and the other to `Violation`.
+. Drag a *DMN Decision* decision node to the canvas.
+. Double-click the decision node and rename it to `Fine`.
+. Click the *Violation* input node, select the *Create DMN Information Requirement* icon and click on the `Fine` decision node to link the two nodes.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/drools-docs/src/main/asciidoc/DMN/dmn-gs-creating-drd-proc.adoc]: It is an icon with this title when hovered over.

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc [hunk 9]
```diff
@@ -2,14 +2,14 @@
 
 = Creating a new *MortgageApprovalProcess* business process application
 
-The following procedures guide you through the creation of each task, connection, and gateway that comprises the *MortgageApprovalProcess* business process.
+The following procedures guide you through the creation of tasks, connections, and gateways that make up the *MortgageApprovalProcess* business process.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: Can you rephrase, I don't understand. There are introductory paragraphs for each procedure.

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc [hunk 10]
```diff
@@ -2,14 +2,14 @@
...
 
 {PRODUCT} contains a predefined selection of node types to simplify business process creation. The predefined node panel is located on the left side of the diagram editor.
 
 image:getting-started/node_panel.png[]
 
-== Validating the mortgage
-The mortgage validation business process determines whether the new application contains the required data before proceeding. If all of the specified data requirements are met, the application moves on to the mortgage calculation business process.
+== Creating the initial mortgage business process
+The mortgage validation business process determines whether a mortgage application contains the required data before the application proceeds. If all of the specified data requirements are met, the application moves on to the mortgage calculation business process.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: Rephrased as:

The mortgage validation business process determines whether a mortgage application contains all required data. If the specified data requirements are met, the application proceeds on to the mortgage calculation business process.

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc [hunk 11]
```diff
@@ -26,77 +26,95 @@ The *Package* specifies the location inside the existing project where the asset
 . Scroll down and expand *Process Data* and click image:getting-started/btn_plus.png[] in the *Process Variables* section.
 . Enter the following values:
 +
-* *Name*: `application`.
+* *Name*: `application`
 * *Data Type*: `Application [com.myspace.mortgage_app]`
 
 == Creating outgoing connections and exclusive gateways
-Create an outgoing connection to an exclusive gateway. Exclusive gateways are used to make decisions and react to events based on the available data.
+This section shows you how to create outgoing connections, exclusive gateways, and business rule tasks. Exclusive gateways are used to make decisions and react to events based on the available data.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: Done

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc [hunk 12]
```diff
@@ -26,77 +26,95 @@ The *Package* specifies the location inside the existing project where the asset
...
 
 .Procedure
-. Click on the start event and create an outgoing connection from the start event to an exclusive gateway.
+
+. Create an outgoing connection from the start event to an exclusive gateway:
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: It is a node icon on the canvas, adding "On the canvas, click the start event node..."

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: As this is part of a getting started guide, I think it doesn't hurt.

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc (hunk 2 above)
Comment (csherrar) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: haha, yeah, first saves the configuration and returns, second is to save the project, and third save confirms.

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc (hunk 3 above)
Comment (csherrar) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: This depends on how the process developer defines the validation data.

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc (hunk 4 above)
Comment (csherrar) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: True, there is no name, adopted suggestion

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc (hunk 9 above)
[the remark this answers is not in the document]
Comment (pnagesha) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: You say "The following procedures..." If you are introducing more than 1 procedure, I am wondering why there is only one introductory paragraph? If this is standard, then just let it be.

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc (hunk 2 above)
Comment (pnagesha) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: Sigh! Ok. Can you specify that the second save is for the project?

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc (hunk 3 above)
Comment (pnagesha) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: So basically the process developer can specify if the validation date  can do any of these functions?

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc (hunk 2 above)
Comment (csherrar) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: Done

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc (hunk 3 above)
Comment (csherrar) [reply doc-content/enterprise-only/getting-started/business-process-proc.adoc]: yes, from my understanding, the developer can create criteria very roughly equivalent to something like these:
loan amount =! null - if false then return loan amount error 
loan amount > 0 - if false then return message "loan amount has to be greater than 0"
date of birth older than (today's date - 18 years) - if false then return message "applicant is under 18"
